refined/shared/src/main/scala/monocle/refined/chars.scala

```diff
@@ -0,0 +1,22 @@
+package monocle.refined
+
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.char.{LowerCase, UpperCase}
+import monocle._
+import monocle.refined.internal.Chars
+
+
+object chars extends CharsInstances
+
+trait CharsInstances {
+  implicit val lowerCase: Prism[Char, LowerCaseChar] = toCase[LowerCase](Chars.lowerCaseChar)
+  implicit val upperCase: Prism[Char, UpperCaseChar] = toCase[UpperCase](Chars.upperCaseChar)
+
+  def toCase[P](implicit C: Chars): Prism[Char, Refined[Char, P]] = {
```

Comment:
would you mind to replace the implicit `Chars` by a function `Char => Boolean`?
e.g. `toCase(p: C => Boolean): Prism[Char, Refined[Char, P]]` and also make it private